src/bundles/SellerRegistration/components/Finish.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 const Finish = ({onClick}) => (
     <div>
-        <h1>That’s it for now!</h1>
+        <h1 tabIndex="-1">That’s it for now!</h1>
```

Comment:
i think you need to describe intent here. maybe the h1 should be wrapped in a hoc from the same file as connectedlink?